gen/x86_table.js

```diff
@@ -621,15 +621,19 @@ const encodings = [
     { sse: 1, opcode: 0x0F10, e: 1, custom: 1 },
     { sse: 1, opcode: 0xF30F10, e: 1, custom: 1 },
     { sse: 1, opcode: 0x660F10, e: 1, custom: 1 },
+    { sse: 1, opcode: 0x660F7C, e: 1, custom: 1 }, // sse3
+    { sse: 1, opcode: 0x660F7D, e: 1, custom: 1 }, // sse3
     { sse: 1, opcode: 0xF20F10, e: 1, custom: 1 },
+    { sse: 1, opcode: 0xF20F7C, e: 1, custom: 1 }, // sse3
```

Comment:
This one is already defined (see the CI failure)

Reply:
it passed make all-debug :D